lib/katello/tasks/upgrades/3.4/reindex_docker_tags.rake

```diff
@@ -0,0 +1,12 @@
+namespace :katello do
+  namespace :upgrades do
+    namespace '3.4' do
+      desc "Reindex docker tags to account for schema 1 and schema 2"
+      task :reindex_docker_tags => ["environment"] do
+        User.current = User.anonymous_admin
+        ::Katello::DockerManifest.import_all
+        ::Katello::DockerTag.import_all
```

Comment:
As a rake task, we can't guarantee that the `DockerManifest` and `DockerTag` classes exist (in the future, say). It's really only important because this will be run during an installer run with `--upgrade` afaik. I seem to remember hitting a similar issue in the past.

Do you think we should define the classes in this task to make the task no-op for that potentially rare case?

Reply:
Wouldnt the unit test that Tom adds fail if these classes dont exist.

Reply:
Yes, that's a great solution. :)